samples/openthread/cli/snippets/ci/ci.conf

```
@@ -16,3 +16,9 @@ CONFIG_SHELL_DEFAULT_TERMINAL_WIDTH=618
 CONFIG_ASSERT=y
 CONFIG_ASSERT_NO_COND_INFO=y
 CONFIG_RESET_ON_FATAL_ERROR=n
+
+# Enable Zperf
+CONFIG_NET_SHELL=y
+CONFIG_NET_ZPERF=y
+CONFIG_NET_CONTEXT_RCVTIMEO=y
+CONFIG_NET_SOCKETS=y
```

```suggestion
CONFIG_NET_SOCKETS=y
CONFIG_NET_SOCKETS_POLL_MAX=9
```
It is now needed for Zperf  APIs

---

done